app/support/page.tsx

```diff
@@ -0,0 +1,72 @@
+'use client'
+import Link from 'next/link';
+import React from 'react';
+
+const handleSave = (e: { preventDefault: () => void; }) =>{
+  e.preventDefault()
+  console.log("save button clicked!")
+}
+
+
+export default function Support() {
+  return (
+    <form className="max-w-lg mx-auto mt-28 p-8">
+      <div className="space-y-12">
+        <div>
+          <h2 className="text-base/7 font-semibold text-gray-900">Support</h2>
+          <p className="mt-1 text-sm/6 text-gray-600">
+          We’re here to support you every step of the way. Just share your email and let us know what’s on your mind—whether it’s a question, a suggestion, or something you need help with. Our team will be in touch soon to make sure you’re taken care of!
+          </p>
+
+          <div className="mt-10 grid grid-cols-1 gap-x-6 gap-y-8 sm:grid-cols-6">
+          <div className="sm:col-span-4">
+              <label htmlFor="email" className="block text-sm/6 font-medium text-gray-900">
+                Email address
+              </label>
+              <div className="mt-2">
+                <input
+                  id="email"
+                  name="email"
+                  type="email"
+                  autoComplete="email"
+                  className="block w-full rounded-md border-0 py-1.5 text-gray-900 shadow-sm ring-1 ring-inset ring-gray-300 placeholder:text-gray-400 focus:ring-2 focus:ring-inset focus:ring-indigo-600 sm:text-sm/6"
+                />
+              </div>
+            </div>
+
+            <div className="col-span-full">
+              <label htmlFor="comment" className="block text-sm/6 font-medium text-gray-900">
+                Comment
+              </label>
+              <p className="mt-3 text-sm/6 text-gray-600">Please share your ideas here.</p>
+              <div className="mt-2">
+                <textarea
+                  id="comment"
+                  name="comment"
+                  rows={3}
+                  className="block w-full rounded-md border-0 py-1.5 text-gray-900 shadow-sm ring-1 ring-inset ring-gray-300 placeholder:text-gray-400 focus:ring-2 focus:ring-inset focus:ring-indigo-600 sm:text-sm/6"
+                  defaultValue={''}
+                />
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+
+      <div className="mt-6 flex items-center justify-end gap-x-6">
+        <button 
```

Comment:
Can I suggest we use the same button for this as the 'Submit' button in the home page? The code for this can be found here:
https://github.com/Servgenie/servgenie-web-frontend/blob/88fcfca840c8cac1817c12863307adedac18385e/components/section/Home/GetReady.tsx#L123

It would be great if we can make this button a reusable component, and pass customized props like 'buttonText' and event handler on click so that we can share the component while reducing code.
<img width="99" alt="Screenshot 2024-11-23 at 10 54 37 AM" src="https://github.com/user-attachments/assets/a7ddeb75-7932-4d49-a46c-25c4adf690db">

Reply:
Once the button component is created, then we can use it here, and update the 2 places below to reuse this component as well.
https://github.com/Servgenie/servgenie-web-frontend/blob/88fcfca840c8cac1817c12863307adedac18385e/components/section/Home/GetReady.tsx#L123
and
https://github.com/Servgenie/servgenie-web-frontend/blob/88fcfca840c8cac1817c12863307adedac18385e/components/section/Home/GetReady.tsx#L123

Reply:
Thank you for the detailed feedback! Did you mean the Support page's submit button for the second use case of the bottom component?

Reply:
That's correct! Once the button component is created, then we can use it here as well.

Reply:
Hi @arihu1008 we've added a commit that consolidates the button that you can use here as well. The component is at `components/common/Button.tsx`